src/libraries/System.Console/tests/TermInfo.Unix.cs

```diff
@@ -103,20 +103,6 @@ public void TermInfoVerification(string termToTest, string expectedForeground, s
         }
     }
 
-    [Fact]
-    [PlatformSpecific(TestPlatforms.AnyUnix)] // Tests TermInfo
-    public void TermInfoE3CanBeRead()
-    {
-        bool ScrollbackBufferIsSet(TerminalData data) => !string.IsNullOrEmpty(data.TerminalDb.ClearScrollbackBuffer);
-
-        // XTerm defines E3 for clearing scrollback buffer and tmux does not.
-        // This can't be added to TermInfoVerification because xterm-256color sometimes has E3 defined (e.g. on Ubuntu but not macOS)
-        XTermData xTermData = new();
-        Assert.True(ScrollbackBufferIsSet(xTermData));
```

Comment:
I don't like the removal of unit test. I would prefer to change the test to verify that `data.TerminalDb.Clear` is given string for these two terminals and keep the test coverage.

Reply:
The unit test isn't really validating anything meaningful, but I can put it back.

Reply:
Please put it back.

Reply:
@stephentoub ping (it will become a stale PR soon)

Reply:
Please feel free to add a test to the PR or merge it as is. I've had other priorities. Thanks.